### PR TITLE
Dratwas/support ssr caching

### PR DESCRIPTION
**Pull request by dratwas**
solves #892

### CODE REVIEW

**Review by zamotany — COMMENTED** [review]
I think we need to document this, right?

File: docs/CachingAndHooks.md [hunk 1]
```diff
@@ -16,6 +16,28 @@ Example:
 *Additional optional cache properties:*
 `cacheTTL` - number of seconds to store the cache for a particular route
 
+Gluestick supprots SSR component caching by using [`electrode-react-ssr-caching`](https://github.com/electrode-io/electrode-react-ssr-caching).
```
Comment (zamotany) [inline docs/CachingAndHooks.md]: `supprots`? 😄

File: packages/gluestick/src/renderer/helpers/cacheManager.js [hunk 2]
```diff
@@ -17,6 +24,13 @@ const getCacheKey = ({ hostname, url }: { hostname: string, url: string}): strin
 };
 
 module.exports = (logger: Logger, isProduction: boolean): CacheManager => {
+  const enableComponentCaching: EnableComponentCaching = (config) => {
+    if (isProduction && !!config && !!config.components) {
```
Comment (zamotany) [inline packages/gluestick/src/renderer/helpers/cacheManager.js]: Are those `!!` really needed?? Both `config` and `config.components` will be evaluated to boolean.

**Review by zamotany — CHANGES_REQUESTED** [review]

File: packages/gluestick/src/renderer/helpers/cacheManager.js (hunk 2 above)
Comment (dratwas) [reply packages/gluestick/src/renderer/helpers/cacheManager.js]: Honestly it's no needed, i was looking at old gluestick :)

**Review by zamotany — APPROVED** [review]

**Pull request merged**